frontend/components/analytics/GoogleAnalytics.tsx

```diff
@@ -0,0 +1,48 @@
+import { GA_URL, GA_KEY } from '@config/env';
+import Head from 'next/head';
+
+export function GoogleAnalytics() {
+   return GA_URL ? (
+      <Head>
+         <script
+            dangerouslySetInnerHTML={{
+               __html: `
+                  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+                  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+                  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+                  })(window,document,'script', '${GA_URL}' ?>','ga');
```

Comment:
Is `?>'` required here or extra left over stuff?

```suggestion
                  })(window,document,'script', '${GA_URL}','ga');
```

Reply:
Looks like it. This was before removing ` ?>'`:
![image](https://user-images.githubusercontent.com/58952979/180890135-e7e9007f-09f9-451d-9da3-e458c974d2ff.png)

This is after removing ` ?>'`:
![image](https://user-images.githubusercontent.com/58952979/180890174-d02b434e-c2f6-474c-aefd-bdfdc72a8ddc.png)